lib/rust/ensogl/core/src/application/shortcut.rs

```diff
@@ -236,6 +250,8 @@ pub struct RegistryModel {
     mouse:              Mouse_DEPRECATED,
     command_registry:   command::Registry,
     shortcuts_registry: shortcuts::HashSetRegistry<Shortcut>,
+    /// Observers that will be notified when a shortcut-triggered command is being executed.
+    observers:          Rc<RefCell<Vec<Observer>>>,
```

Comment:
Cannot we just have an `frp::Source` to which every observer is attached?

It could be even public, so in the usage place you won't need any intermediate node - just
```rust
frp.source.current_shortcut <+ self.model.app.shortcuts.current_shortcut;
```